department-of-veterans-affairs/va.gov-cms
# VACMS-2312: FAQ - multiple Q&A content type and component

## Description

See #2312 . 

includes:
*  `FAQ - multiple Q&As` content type
* minor updates to `Q&A` content type and Step-by-Step content type

## Screenshots

### "Q&A" form (single)
![Q A single](https://user-images.githubusercontent.com/16477724/89039653-af6ef780-d2ff-11ea-96df-15a9647dc98a.png)


### "FAQ - multiple Q&As" form
![FAQ-multiple](https://user-images.githubusercontent.com/16477724/89036742-73856380-d2fa-11ea-80b8-3293acb0852b.png)

### "Step-by-step" form
![ste-by-step](https://user-images.githubusercontent.com/16477724/89036748-75e7bd80-d2fa-11ea-8d4f-765e95c9b183.png)

## QA steps

- [x] review "FAQ - multiple Q&As" form - visit http://pr2511.ci.cms.va.gov/node/add/faq_multiple_q_a
    - [x] confirm fields include
         - Title and introduction fieldset: Page title and Page introduction (field_intro_html_limited)
         - Meta tags fieldset : Title and Description
         - Meta tags fieldset in the sidebar is not displayed on the form
         - CTA buttons field that allows for up to 2 max buttons to be included
         - INCLUDE TABLE OF CONTENTS? fieldset
         - Q&A group field that allow to add groups of Q&As w/ a required header for each group
         - "Repeat CTA buttons" checkbox
         - revision log message field is displayed within Editorial Workflow fieldset
         - owner fields displayed in sidebar
    
- [x] review "Q&A" form - visit http://pr2511.ci.cms.va.gov/node/add/q_a
    - [x] confirm fields include
         - Question and Answer fieldset: Question and Answer fields
             - Answer field is limited to WYSIWYG or Alert components
         - Meta tags fieldset : Title and Description
         - Meta tags fieldset in the sidebar is not displayed on the form
         - CTA buttons field that allows for up to 2 max buttons to be included
         - revision log message field is displayed within Editorial Workflow fieldset
         - owner field displayed in sidebar
         
- [x] review "Step-by-step" form _updates_ - visit http://pr2511.ci.cms.va.gov/node/add/step_by_step
    - [x] confirm fields include
         - Title and introduction fieldset: Page title and Page introduction (field_intro_html_limited)
         - Meta tags fieldset in the sidebar is not displayed on the form
         - revision log message field is displayed within Editorial Workflow fieldset
         - owner field displayed in sidebar


NOTE: @paragraph_fields behat test is disabled temporarily so that we can deliver content model updates requested by PW team in this sprint. Follow up defect is opened here - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2533
